metagraph/src/graph/alignment/dbg_aligner.hpp

```diff
@@ -14,6 +14,9 @@
 
 
 namespace mtg {
+
+using mtg::common::logger;
```

Comment:
In general, yes. The problem in this specific case is that I didn't want to put the implementations of the `DBGAligner` methods into a cpp file to avoid having a bunch of instantiations for each combination of Seeder and Extender.

Reply:
then you shouldn't use `mtg::common::`, I guess